web/themes/new_weather_theme/assets/sass/components/obs-table.scss

```diff
@@ -0,0 +1,23 @@
+@use "uswds-core" as *;
+
+/* Observation table overrides
+------------------------------- */
+.usa-table.observation-table {
+  td,
+  th {
+    padding-left: 0;
+    vertical-align: top;
+    padding-top: units(1) units(4) units(1) inherit;
```

Comment:
I think the property here is meant to be `padding` not `padding-top`

For the `th` padding-left value could we set it to 0? 

And then for the padding-bottom value, the `usa-table` line-height makes this one a little more tricky to set. If possible could we set the line-height for this table to line height token `2` (should hopefully be calculated as `1.27`), and then set the padding-bottom value here to `units(0.5)`? Hoping to achieve tighter, and even-ish spacing like here:

![Screenshot of adjust spacing for the variable table](https://github.com/weather-gov/weather.gov/assets/150970973/75df75b2-2de1-464b-b2bf-28175490dde9)

Reply:
https://github.com/weather-gov/weather.gov/pull/1285/commits/12281022d4a3b21167568782ba47e359ee8f1fa1